pinc/faq.inc

```diff
@@ -97,7 +97,8 @@ $Stealth_Scannos_URL = make_forum_url("t", $Stealth_Scannos_id, 1563);
 $Proofing_Interface_Help_URL = make_forum_url("t", $Proofing_Interface_Help_id, 2739);
 $XHTML_suggestion_URL = make_forum_url("t", $XHTML_suggestion_id, 4405);
 
-function make_forum_url($type, $id, $default)
+/** @param string|int $id */
```

Comment:
We don't have typing for all types of URL we generate below so I sticked to this type for now, though we may be able to narrow this down to `int` down the road :crossed_fingers: